boards/arm/nrf51dongle_nrf51422/doc/index.rst

```diff
@@ -1,7 +1,7 @@
-.. _nrf51_pca10031:
+.. _nrf51dongle_nrf51422:
 
-nRF51-PCA10031
-##############
+nRF51-Dongle
```

Comment:
"-" is not really needed here (we don't have this in the other Doc titles)

Reply:
Done